applications/ATOM/tools/compute_vocab.cpp

```diff
@@ -0,0 +1,99 @@
+#include <iostream>
+#include <string>
```

Comment:
I suggest we rename _tools_ directory as _utils_ directory. I have third party ("sophisticated") python scripts as well. Unlike this scripts that is standalone, mine are called from python frontend so I place them in utils directory. It is fine with me if choose to retain _tools_ directory, just a comment.